core-java/src/main/java/eu/neverblink/jelly/core/JellyConverterFactory.java

```diff
@@ -0,0 +1,120 @@
+    }
+
+    /**
+     * Create a new QuadsDecoder.
+     * @param supportedOptions maximum supported options for the decoder. If not provided, this.defaultSupportedOptions
+     *                         will be used. If you want to modify this (e.g., to specify an expected logical stream
+     *                         type), you should always use this.defaultSupportedOptions.withXxx.
+     * @param quadProtoHandler the handler to use for decoding quads
+     * @return decoder
+     */
+    public final ProtoDecoder<TNode, TDatatype> quadsDecoder(
+        RdfHandler.QuadHandler<TNode> quadProtoHandler,
+        RdfStreamOptions supportedOptions
+    ) {
+        return new ProtoDecoderImpl.QuadsDecoder<>(decoderConverter(), quadProtoHandler, supportedOptions);
+    }
+
+    /**
+     * Create a new GraphsAsQuadsDecoder.
+     * @param supportedOptions maximum supported options for the decoder. If not provided, this.defaultSupportedOptions
+     *                         will be used. If you want to modify this (e.g., to specify an expected logical stream
+     *                         type), you should always use this.defaultSupportedOptions.withXxx.
+     * @param graphProtoHandler the handler to use for decoding graphs
+     * @return decoder
+     */
+    public final ProtoDecoder<TNode, TDatatype> graphsAsQuadsDecoder(
+        RdfHandler.QuadHandler<TNode> graphProtoHandler,
+        RdfStreamOptions supportedOptions
+    ) {
+        return new ProtoDecoderImpl.GraphsAsQuadsDecoder<>(decoderConverter(), graphProtoHandler, supportedOptions);
+    }
+
+    /**
+     * Create a new GraphsDecoder.
+     * @param supportedOptions maximum supported options for the decoder. If not provided, this.defaultSupportedOptions
+     *                         will be used. If you want to modify this (e.g., to specify an expected logical stream
+     *                         type), you should always use this.defaultSupportedOptions.withXxx.
+     * @param graphProtoHandler the handler to use for decoding graphs
+     * @return decoder
+     */
+    public final ProtoDecoder<TNode, TDatatype> graphsDecoder(
+        RdfHandler.GraphHandler<TNode> graphProtoHandler,
+        RdfStreamOptions supportedOptions
+    ) {
+        return new ProtoDecoderImpl.GraphsDecoder<>(decoderConverter(), graphProtoHandler, supportedOptions);
+    }
+
+    /**
+     * Create a new AnyStatementDecoder.
+     * @param supportedOptions maximum supported options for the decoder. If not provided, this.defaultSupportedOptions
+     *                         will be used. If you want to modify this (e.g., to specify an expected logical stream
+     *                         type), you should always use this.defaultSupportedOptions.withXxx.
+     * @param anyProtoHandler the handler to use for decoding any statements
+     * @return decoder
+     */
+    public final ProtoDecoder<TNode, TDatatype> anyDecoder(
```

Comment:
anyStatementDecoder...


because it produces only triples and quads, never graph events